.gitlab-ci.yml

```diff
@@ -0,0 +1,13 @@
+image: node:14
```

Comment:
does this get replaced with similar on github ci?

Reply:
Wasn't paying enough attention, just copied over the gitlab config. will update for github.